### PR TITLE
Fix documentation issues

### PR DESCRIPTION
**Pull request by saschagrunert**
I came across this project and found possible enhancements to the documentation:

- mention that the release binaries can be used directly
- added the `pre-build` to Cross.toml `build` section, which also works
  for all targets
- fixed the architecture in the arm64 example in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ default to `docker`.
 cargo install cross --git https://github.com/cross-rs/cross
 ```
 
+It's also possible to directly download the pre-compiled [release
+binaries](https://github.com/cross-rs/cross/releases).
+
```
Comment (Emilgardis) [inline README.md:63]: can we have a little mention to https://github.com/cargo-bins/cargo-binstall here maybe?? what do you think?

**Review by Emilgardis — APPROVED** [review]
Awesome, thanks for finding these improvements :D

looks good, a small comment I think we could address but otherwise this is just fine

File: README.md (hunk 1 above)
Comment (saschagrunert) [reply README.md]: Yes why not, let's mention it as well :)

**Review by Alexhuszagh — APPROVED** [review]
Great finds.
bors r=Emilgardis,Alexhuszagh

**Comment by Emilgardis** [comment]
lets try again

bors r=Emilgardis,Alexhuszagh

**Pull request merged**